frontend/src/cards/MapCard.tsx

```diff
@@ -405,6 +408,10 @@ function MapCardWithKey(props: MapCardProps) {
         const mapConfig = props.dataTypeConfig.mapConfig
         if (isSummaryLegend) mapConfig.min = mapConfig.mid
 
+        const updatedMapConfig = Object.assign({}, mapConfig, {
+          scheme: interpolateDarkGreen,
+        })
+
```

Comment:
this is making every map the green scale, regardless of the type of scale we assigned to that dataType in the metric config. I think it's partly confusing as well because you have made a new `MapConfig` type that has a function type as the `scheme` and the old `MapConfig` has a string as the `scheme`. I would use a unique name for the new type like `D3MapConfig` and then figure out is we can make the types identical so everything plays nice

Reply:
But I dont think this override should be happening here... the MapCard should ideally be left as is, and then within the new ChoroplethMap it should be retrieving the specific map scale colors from the MetricConfig